src/general/Mutations.cs

```diff
@@ -170,96 +170,132 @@ private void MutateBehaviour(Species parent, Species mutated)
     {
         var nucleus = SimulationParameters.Instance.GetOrganelleType("nucleus");
 
-        for (var iteration = 0; iteration < 10; iteration++)
+        mutatedOrganelles.Clear();
```

Comment:
Is this still needed without iterations to wipe away?

Reply:
This list is provided as parameter and needs to be cleared here, as it contains the copied organelles from the parent.

Reply:
We provide a list as a param, specifically to clear it out? Sounds like we can just drop the param and declare the variable in here.

Reply:
It's easier to pass in the mutated list of organelles. As the mutated species is created by cloning, so it would actually be more work to specifically remove the organelles from it to process here.